python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_traffic_engineering.schema.yml

```diff
@@ -91,3 +91,65 @@ keys:
                                     type: int
                                     convert_types:
                                     - "str"
+      flex_algos:
+        type: list
+        items:
+          type: dict
+          keys:
+            number:
+              type: int
+              required: true
```

Comment:
Should this instead be a `primary_key` on the list, if it is supposed to be unique?

Reply:
Also would name be unique as well? Or is it only the combination of name and number that must be unique?

Reply:
Both name and number must be unique across all flexalgo definitions, added primary and unique keys to schema. Thanks!